<launch>
  
   <node name ="panda_joint_reader_node" pkg="panda_joint_reader" type= "panda_joint_reader_node" output="screen" >
      <rosparam command="load" file="$(find panda_joint_reader)/config/param.yaml" />
   </node>


</launch>